### PR TITLE
add additional valid checksum for lhs 1.1.1 extension in recent R 4.0.x and 4.1.0 easyconfigs

### PR DESCRIPTION
**Pull request by guimou**
When trying to build R 4.1.0 I found that the checksum for the lhs v1.1.1 extension was wrong.
I corrected with the right checksum (build tested and working) for all its occurences.
Don't know what happened, maybe they repackaged/replaced the source at some point...

### CODE REVIEW

**Comment by boegel** [comment]
This isn't the first time, and it won't be the last time...

As usual, there's actually no code changes at all, the only difference in the files included in the new source tarball is the publication date:

```
$ diff -ru lhs_1.1.1_old lhs_1.1.1_new
diff -ru lhs_1.1.1_old/DESCRIPTION lhs_1.1.1_new/DESCRIPTION
--- lhs_1.1.1_old/DESCRIPTION   2020-10-05 11:50:14.000000000 +0200
+++ lhs_1.1.1_new/DESCRIPTION   2021-09-07 20:54:37.000000000 +0200
@@ -23,4 +23,4 @@
 Author: Rob Carnell [aut, cre]
 Maintainer: Rob Carnell <bertcarnell@gmail.com>
 Repository: CRAN
-Date/Publication: 2020-10-05 09:50:14 UTC
+Date/Publication: 2020-10-05 11:50:16
diff -ru lhs_1.1.1_old/MD5 lhs_1.1.1_new/MD5
--- lhs_1.1.1_old/MD5   2020-10-05 11:50:14.000000000 +0200
+++ lhs_1.1.1_new/MD5   2021-09-07 20:54:37.000000000 +0200
@@ -1,5 +1,5 @@
 3974924d50dfb5a3b52272d2186ddfd2 *ChangeLog
-db7fc2acd602c9794addccad90c99819 *DESCRIPTION
+95d7aaec3a50188e31528e495a907426 *DESCRIPTION
 190f0a824ae4601a8774a1e687da96c1 *NAMESPACE
 53866b3921acf88d621f1b67d6bf5517 *R/augmentLHS.R
 2e25e3a6e4b33b3d25035948770bd0ea *R/createOA.R
```

Therefore, it makes sense to keep accepting both the old and the new SHA256 checksum, since it doesn't really matter which source tarball you use.

File: easybuild/easyconfigs/r/R/R-4.1.0-foss-2021a.eb [hunk 1]
```diff
@@ -1549,7 +1549,7 @@ exts_list = [
         'checksums': ['0bd2dcd4ee1915448d325fb5e66bee68e0497cbd91ef67a11b400b2fbe52ff59'],
     }),
     ('lhs', '1.1.1', {
-        'checksums': ['903e9f2adde87f6f9ad41dd52ff83d28a645dba69934c7535142cb48f10090dc'],
+        'checksums': ['00278a65b18c86799301d27c55e3d32c6c307aed745ea69c9b96bbb1d0a352c2'],
```
Comment (boegel) [inline easybuild/easyconfigs/r/R/R-4.1.0-foss-2021a.eb:1552]: @guimou Please (consistently) change to accept *both* checksums as valid (like we're already doing

```suggestion
        'checksums': [('903e9f2adde87f6f9ad41dd52ff83d28a645dba69934c7535142cb48f10090dc',
                       '00278a65b18c86799301d27c55e3d32c6c307aed745ea69c9b96bbb1d0a352c2')],
```

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/r/R/R-4.1.0-foss-2021a.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/r/R/R-4.1.0-foss-2021a.eb]: I've applied this fix myself in 64677cab42, so we can get this PR merged quickly...

**Review by boegel — APPROVED** [review]

File: easybuild/easyconfigs/r/R/R-4.1.0-foss-2021a.eb (hunk 1 above)
Comment (guimou) [reply easybuild/easyconfigs/r/R/R-4.1.0-foss-2021a.eb]: Lol, thanks a lot!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in total)
node3302.joltik.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz (cascadelake), Python 3.6.8
See https://gist.github.com/374749d8f4969e43f125eb5ce1d3f4dc for a full test report.

**Comment by boegel** [comment]
Going in, thanks @guimou!

**Pull request merged**